ambry-clustermap/src/main/java/com/github/ambry/clustermap/HelixParticipant.java

```diff
@@ -896,6 +896,10 @@ public void onPartitionBecomeBootstrapFromOffline(String partitionName) {
       if (statsManagerListener != null) {
         statsManagerListener.onPartitionBecomeBootstrapFromOffline(partitionName);
       }
+    } catch (InterruptedException e) {
+      logger.error("Bootstrap was interrupted on partition {}", partitionName);
```

Comment:
Can we add more details in the error log?

Reply:
added a todo.